apps/sertra/components/ServerList.tsx

```diff
@@ -64,24 +53,20 @@ export const ServerList: React.FC<PropsWithChildren<ServerListProps>> = ({
   }, [community]);
 
   return (
-    <div>
-      <div className={styles.header}>
-        <div className={styles.titleContainer}>
-          <div className={styles.pageTitle}>Servers</div>
-          <div className={styles.gameTitle}>{gameDisplayName}</div>
-        </div>
+    <>
+      <div className={styles.titleContainer}>
+        <div className={styles.pageTitle}>Servers</div>
+        <div className={styles.gameTitle}>{gameDisplayName}</div>
       </div>
       <div className={styles.tableHeader}>
```

Comment:
IMO it would be beneficial to wrap all of the listing items in a container.

Reply:
Same comment as above. Not against it, but I'd like to know why you think that?

Reply:
Same as above. In addition, as an example here we could leverage flexbox + gap, leaving the margin-bottom out of the children.